components-core/src/main/java/com/adyen/checkout/components/api/ImageLoader.kt

```diff
@@ -43,10 +43,40 @@ class ImageLoader(private val logoApi: LogoApi) {
      * Load image to ImageView with place holder before load and error fallback image.
      */
     @JvmOverloads
+    fun load(
+        txVariant: String,
+        view: ImageView,
+        size: LogoApi.Size?,
+        @DrawableRes placeholder: Int = 0,
+        @DrawableRes errorFallback: Int = 0
+    ) {
+        load(txVariant, "", view, size, placeholder, errorFallback)
+    }
+
+    /**
+     * Load image to ImageView with place holder before load and error fallback image.
+     */
+    @JvmOverloads
+    fun load(
+        txVariant: String,
+        txSubVariant: String,
+        view: ImageView,
+        @DrawableRes placeholder: Int = 0,
+        @DrawableRes errorFallback: Int = 0
+    ) {
+        load(txVariant, txSubVariant, view, null, placeholder, errorFallback)
```

Comment:
Should we keep the default size as null for existing usages? Just making sure we should have been passing it before and we weren't

Reply:
I kept it as null since that was the previous implementation. But I see your point about passing the default size here rather than having to leave that logic to `LogoApi`.